b2c_authentication.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+/**
+ * Plugin Name: b2c_authentication
```

Comment:
>b2c_authentication [](start = 16, length = 18)

This should be a more descriptive name. (It does not have to be the same as the pugin's differentiation slug (e.g. "`b2c`", from the function names below).)